### PR TITLE
Home work week3 added

### PR DESCRIPTION
**Pull request by Hani-far**

### CODE REVIEW

File: week3/Week3HomeWork/Controllers/ConverterController.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+using Week3HomeWork.Models;
+
+namespace Week3HomeWork.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ConverterController : ControllerBase
+    {
+        [HttpGet("GallonsToLitres")]
+        public double GallonsToLitres(int Litres)
```
Comment (calindurnea) [inline week3/Week3HomeWork/Controllers/ConverterController.cs:16]: Name of variables and parameters should start with lowercase

File: week3/Week3HomeWork/Controllers/ConverterController.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        [HttpGet("ConvertMiles")]
+        public Distance ConvertMiles(int miles)
+        {
+            return new Distance { Miles = miles, Kilometers = miles * 1.609 };
+        }
+
+        [HttpPost("convert")]
+        public List<ConversionResponse> ConvertValues([FromBody] ConversionRequest Request)
+        {
+            return new ConversionResponse().conversions(Request);
```
Comment (calindurnea) [inline week3/Week3HomeWork/Controllers/ConverterController.cs:30]: Method names should start with uppercase, `Conversions` in this case, and variables with lowercase `request`

File: week3/Week3HomeWork/Week3HomeWork.csproj [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```
Comment (calindurnea) [inline week3/Week3HomeWork/Week3HomeWork.csproj:4]: Are you intentionally using the net5.0 version here? 😊

**Review by calindurnea — COMMENTED** [review]
Great job with the tasks! 👍 
I left a few comments, let me know if you have any questions

File: week3/Week3HomeWork/Controllers/ConverterController.cs (hunk 1 above)
Comment (Hani-far) [reply week3/Week3HomeWork/Controllers/ConverterController.cs:16]: Done

File: week3/Week3HomeWork/Controllers/ConverterController.cs (hunk 2 above)
Comment (Hani-far) [reply week3/Week3HomeWork/Controllers/ConverterController.cs:30]: Done

File: week3/Week3HomeWork/Week3HomeWork.csproj (hunk 3 above)
Comment (Hani-far) [reply week3/Week3HomeWork/Week3HomeWork.csproj:4]: Yes, I have done it with my work computer which has a net5.0 version. :)

**Comment by Hani-far** [comment]
> Great job with the tasks! 👍 I left a few comments, let me know if you have any questions

**Pull request merged**